cloudfront-s3-cdk-go/example-pattern.json

```diff
@@ -0,0 +1,56 @@
+{
+    "title": "CloudFront Distribution in front of S3 Static Website",
+    "description": "Create an Amazon S3 Static Website with a CloudFront Distribution in front of it.",
+    "language": "Go",
+    "level": "200",
+    "framework": "CDK",
+    "introBox": {
+      "headline": "How it works",
+      "text": [
+        "This pattern shows how to Create an Amazon S3 Static Website with a CloudFront Distribution in front of it. Implemented in AWS CDK using the Go programming language.",
+        "This pattern deploys one CloudFront Distribution and an S3 Static Website."
+      ]
+    },
+    "gitHub": {
+      "template": {
```

Comment:
The projectFolder is the required field.  Also since the cloudfront-s3-cdk-go.go was renamed, its throwing a new error. Can you rename this file to cloudfronts3-cdk-go.go?

Can you please update as below

```
        "template": {
            "repoURL": "https://github.com/aws-samples/serverless-patterns/tree/main/cloudformation-s3-cdk-go",
            "templateURL": "serverless-patterns/cloudformation-s3-cdk-go",
            "projectFolder": "cloudfront-s3-cdk-go",
            "templateFile": "cloudfronts3-cdk-go.go"
        }
```